dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -74,9 +76,8 @@
         {% do exceptions.raise_compiler_error(inc_predicates_not_list) %}
       {% endif %}
     {% endif %}
-    {% set tmp_relation = make_temp_relation(target_relation) %}
```

Comment:
any specific reason why this was moved on top?

Reply:
It was re-assigned multiple times to `make_temp_relation(target_relation)`, and never used the original assignment, I thought it was a clean and easy refactor